src/UI/App.jsx

```diff
@@ -562,7 +562,10 @@ export default class App extends React.Component {
 			<InterfaceTextsContext.Provider value={this.state.interfaceTexts}>
 				{
 					!(this.state.offline && this.state.hideChatOutsideWorkingHours)
-						&& <Launcher onClick={this.handleClick} />
+						&& <Launcher
+							messengerState={this.state.showChat ? this.messengerStateOpen : this.messengerStateMinimize}
```

Comment:
Waarsch kan ik nu rechtstreeks met localstorage de waarde ophalen. (omdat ik dit nu al componentDidMount controleer

Reply:
in afwachting van ticket: Add messenger open state #39